README.md

```diff
@@ -1,21 +1,97 @@
+# Compile instruction
+
+## Before compiling
+
+please check the [Releases](https://github.com/Franklalalala/usenauty/releases). There are pre-compiled linux and windows versions, respectively.
+
```

Comment:
Have you sure the runtime library version? Or is it compile static and lowest-system rt libraries?